internal/kademlia/kademliaid/calc_distance.go

```diff
@@ -0,0 +1,29 @@
+package kademliaid
+
+import (
+	"fmt"
+)
+
+// CalcDistance returns a new instance of a KademliaID that is built
+// through a bitwise XOR operation between kademliaID and target
+func (kademliaID KademliaID) CalcDistance(target KademliaID) KademliaID {
+	byteSlice, _ := bitwiseXOR(kademliaID.Bytes(), target.Bytes())
```

Comment:
should we handle this error?

Reply:
I split the function for testing reasons.
Two kademlia ids always have the same length, so I think it cannot fail. 
What do u think?

Reply:
Yeah, that's true. This should be fine then :D